web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js

```diff
@@ -0,0 +1,26 @@
+var AuditorMouseMovementSpecific = {
+    mouse_movement_specific: [],
+    log_mousemove_specific: function (e) {
+            var position = [e.pageX, e.pageY];
+            this.mouse_movement_specific.push(position);
+        }
+    },
+    submit_callable: function () {
+        return {
+            'content':  this.mouse_movement_specific
+        };
+    }
+};
```

Comment:
Same basic thing here. This javascript can be mapped a lot closer to what models will look like and that will save you and I a lot of code/pain (especially if we end up having to change parts of how Models work and we can just modify parents without worrying about downstream code.